<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharCreateWindow</class>
 <widget class="QDialog" name="CharCreateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_13">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_name">
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Name :</string>
         </property>
         <property name="buddy">
          <cstring>field_name</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="field_name">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_class">
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Class :</string>
         </property>
         <property name="buddy">
          <cstring>field_class</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="field_class">
         <property name="minimumSize">
          <size>
           <width>145</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>145</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
setItemData(i, Qt::AlignCenter, Qt::TextAlignmentRole);
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_gender">
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Gender:</string>
         </property>
         <property name="buddy">
          <cstring>field_race</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="field_gender">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>145</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>145</width>
           <height>25</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
setItemData(i, Qt::AlignCenter, Qt::TextAlignmentRole);
}</string>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_race">
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Race :</string>
         </property>
         <property name="buddy">
          <cstring>field_race</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="field_race">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>145</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>145</width>
           <height>25</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
setItemData(i, Qt::AlignCenter, Qt::TextAlignmentRole);
}</string>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="labeL_alignment">
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Alignment :</string>
         </property>
         <property name="buddy">
          <cstring>field_alignment</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="field_alignment">
         <property name="minimumSize">
          <size>
           <width>145</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>145</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
setItemData(i, Qt::AlignCenter, Qt::TextAlignmentRole);
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_background">
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Background :</string>
         </property>
         <property name="buddy">
          <cstring>field_background</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="field_background">
         <property name="minimumSize">
          <size>
           <width>145</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>145</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
setItemData(i, Qt::AlignCenter, Qt::TextAlignmentRole);
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_balance">
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Balance :</string>
         </property>
         <property name="buddy">
          <cstring>field_balance</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="field_balance">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_level">
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Level :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>field_level</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="field_level">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="placeholderText">
          <string>1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_spawnat">
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Spawn at :</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_x">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>ImperatorSmallCaps</family>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>X :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
             <property name="buddy">
              <cstring>field_y</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="field_x">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>25</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>ImperatorSmallCaps</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="placeholderText">
              <string>25</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_y">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>17</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>ImperatorSmallCaps</family>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Y :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
             <property name="buddy">
              <cstring>field_x</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="field_y">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>ImperatorSmallCaps</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QPushButton" name="pushButton_confirm">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Confirm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_cancel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ImperatorSmallCaps</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
